src/agents/triggers/trigger.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * All triggers are invisible entities that cause the game engine to
+ * act in an unique way.
+ */
+function Trigger(game, x, y, width, height) {
```

Comment:
If this kind of Trigger is not intended to actually be used directly, it would be a good idea to rename it "BaseTrigger" or something similar.